.github/workflows/build.yml

```diff
@@ -1,27 +1,22 @@
 name: build
 
-on: [push, pull_request]
+on: 
+  push:
+    branches: [main]
+  pull_request:
 
 jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.5, 2.6, 2.7, "3.0", 3.1, head]
-        gemfile: [rails_5_2, rails_6]
-        exclude:
-          - ruby: "3.0"
-            gemfile: rails_5_2
-          - ruby: 3.1
-            gemfile: rails_5_2
-          - ruby: head
-            gemfile: rails_5_2
-    continue-on-error: ${{ endsWith(matrix.ruby, 'head') }}
```

Comment:
Are these changes related to ignoring sqlite?

Reply:
Hmmm, they're not. I'll clean it up quick.